influxql/engine.go

```diff
@@ -314,7 +314,7 @@ loop:
 	for _, row := range rows {
 		for _, values := range row.Values {
 			t := time.Unix(0, values[0].(int64))
-			values[0] = t.UTC().Format(time.RFC3339Nano)
+			values[0] = t.UTC()
```

Comment:
We should allow storing time as a field value, but it's not a priority for now. Right now it's just int, float64, bool, string, and bytes